services/src/datasets/ml_model.rs

```diff
@@ -0,0 +1,23 @@
+use async_trait::async_trait;
```

Comment:
move this file to the machine_learning folder

Reply:
d558b40a

---

Commit MSG:

refactor: move ml_model.rs from services/src/datasets to services/src/machine_learning